changelog/0032-upgrade-elasticsearch.yml

```diff
@@ -0,0 +1,84 @@
+---
+title: ES upgrade from 1.7.6 to 2.4.6
+key: upgrade-elasticsearch
+date: 2020-02-05
+optional_per_env: no
+min_commcare_version: 19be50699bf78b25a8da6e3e13eb5bc6d13245e8
```

Comment:
I think this is the wrong commit, looks like it was copied from `0030-upgrade-sentry.yml`. You can use 064bb6dfc2b333cc8e6e3955b0c5d30513c5f4be instead.